__tests__/language/language.reducer.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef */
+import Language from '../../src/redux/language/language.reducer';
+import { CHANGE_LANGUAGE } from '../../src/redux/language/language.types';
+
+describe('test language reducer', () => {
+  test('should return default state', () => {
+    expect(Language()).toMatchSnapshot();
+    expect(Language()).toEqual({
+      language: 0
+    });
+  });
+  test('should change language', () => {
+    expect(Language({ language: 1 })).toMatchSnapshot();
+    expect(Language({ language: 1 })).toEqual({
```

Comment:
Reducer should take two arguments - currentState and action, here it takes only one. The test works but doesn't test anything